AUTHORS.md

```diff
@@ -0,0 +1 @@
+# Authors
```

Comment:
What do we do here?

Reply:
I’d just say something like “maintained by the Trixi.jl authors” and refer to the Trixi.jl authors.md file.